apps/www/src/content/blog/import-from-obsidian.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Importing Content From Obsidian
+featuredImageUrl: https://static.feynote.com/screenshots/blog-import-obsidian-20251203.jpg
+description: A step-by-step guide for importing your content from obsidian into FeyNote.
+preview: A step-by-step guide for importing your content from obsidian into Feynote.
+readingTime: 3 Minute Read
+publishDate: 2025-12-03
+updatedDate: 2025-12-03
+---
+
+Importing your content from Obsidian can be done in a few simple steps.
+
+---
+
+### 1. Open Obsidian and click the ellipsis menu
+
+When Obsidian launches, look for the **three-dot (ellipsis) menu** at the top-right corner of the application.
+
+<div style="text-align: center; margin: 1.5em 0;">
+  <img
+    src="https://static.feynote.com/screenshots/import-instructions/obsidian/three-dots-click-20250527.png"
+    alt="Cursor hovering the ellipsis button in the top right of your Obsidian application"
+    width="600"
+    style="border-radius: 6px;"
+  />
```

Comment:
See this file for how to do responsive optimized images within Astro (note .mdx file instead of .mx)

https://raw.githubusercontent.com/FeyNote/FeyNote/refs/heads/main/apps/www/src/content/blog/test-post-1.mdx 

It can be done with .md files but must be using a markdown image tag which doesn't allow styling.